lib/vector-core/src/schema/definition.rs

```diff
@@ -15,11 +15,15 @@ use value::{
 #[derive(Clone, Debug, PartialEq, PartialOrd)]
 pub struct Definition {
     /// The type of the event
-    kind: Kind,
+    event_kind: Kind,
+
+    /// The type of the metadata
```

Comment:
```suggestion
    /// The type of the metadata.
```

(The comment above that is also missing a `.`)